src/dom.ts

```diff
@@ -116,7 +116,7 @@ export function contains(parent: Element, node: Node): boolean {
  */
 
 export let create: CreateFunction = function (
-		tagName: string, kwArgs?: CreateArgs, children?: (Node | string)[]): HTMLElement {
+		tagName: string, kwArgs?: CreateArgs, children?: (Node | string)[]): any {
```

Comment:
The interface has multiple overrides with more specific return types such as `HTMLAnchorElement`, with the TS2 compiler this existing implementation fails for `'HTMLElement' is not assignable to type ‘HTMLAnchorElement’`.